docs/dev/test-configuration-options.md

```diff
@@ -1965,3 +1965,9 @@ These are currently not supported for real devices:
 - `removeApp`: Managed by RDC differently, but cannot be used inside an Appium test as part of the routine.
 - `Edit Timezone`: Appium does not provide a capability to edit the timezone of a device in automated testing on real devices.
 - See [Virtual Device Capabilities](#virtual-device-capabilities-sauce-specific--optional) for information about timezone capabilities in a virtual device testing.
+
+## Limitations
+
+:::warning
+When running a test on a Virtual Device, please be aware that there is a 100 characters limitation for capabilities value. If the value exceeds this limit, it will be truncated and leading to issues. Make sure the value does not exceed the limitation to avoid any issues.
```

Comment:
```suggestion
When running a test on a Virtual Device, please be aware that there is a 100 characters limitation for each capability value. If the value exceeds this limit, it will be truncated, which can lead to further side effects or prevent a job from starting.
```